lib/network.go

```diff
@@ -6099,11 +6400,23 @@ func (tsl *TransactionSpendingLimit) Copy() *TransactionSpendingLimit {
 		copyTSL.AccessGroupMemberMap[accessGroupMemberLimitKey] = accessGroupMemberCount
 	}
 
+	for stakeLimitKey, stakeLimitDESONanos := range tsl.StakeLimitMap {
+		copyTSL.StakeLimitMap[stakeLimitKey] = stakeLimitDESONanos
+	}
+
+	for stakeLimitKey, unstakeLimitDESONanos := range tsl.UnstakeLimitMap {
+		copyTSL.UnstakeLimitMap[stakeLimitKey] = unstakeLimitDESONanos
```

Comment:
We should copy the uint256s - we don't need to copy the uint64s since they aren't pointers.
```suggestion
		copyTSL.StakeLimitMap[stakeLimitKey] = stakeLimitDESONanos.Clone()
	}

	for stakeLimitKey, unstakeLimitDESONanos := range tsl.UnstakeLimitMap {
		copyTSL.UnstakeLimitMap[stakeLimitKey] = unstakeLimitDESONanos.Clone()
```